sample.env

```diff
@@ -48,3 +48,13 @@ WATCH_EPICGAMES=false
 
 # Comma-separated Applications IDs to track.
 EPICGAMES_APP_IDS="963137e4c29d4c79a81323b8fab03a40, bcbc03d8812a44c18f41cf7d5f849265, Kinglet"
+
+# GOG ------------------------------------------------
+# Set "true" if you want to track the products on GOG.
+WATCH_GOG=false

```

Comment:
Could you please add new envs (`WATCH_GOG` ans `GOG_APP_IDS`) to `docker-compose.yml`?

Reply:
Done